components/homepage-contributor-spotlight/server.css

```
@@ -79,6 +79,7 @@
     place-self: center;
     width: max-content;
     max-height: 8rem;
+    object-fit: contain;
```

To fix the stylelint issues:

```suggestion
    place-self: center;

    width: max-content;
    max-height: 8rem;

    object-fit: contain;
```

When committing locally, that should get fixed via lefthook (using the pre-commit hook).